src/WebCompiler/Config/Config.cs

```diff
@@ -62,6 +76,19 @@ public class Config
 
         internal string Output { get; set; }
 
+
+        /// <summary>
+        /// Determines if the config is only an extension pattern - not real file to process.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsExtensionPattern => this.InputFile?.StartsWith("*") ?? false;
```

Comment:
Could this instead just search anywhere for the wildcard character?  Just thinking of cases like /MyFolder/*.less